GameData/SPEngine/Families/n2.cfg

```diff
@@ -32,7 +32,7 @@ SPEFamily:NEEDS[RealFuels]
 		maxIgnitions = 0
 		mass = 0.452
 		entryCost = NK-Family
-		cost = 635
+		cost = 500
```

Comment:
Care to add comments to these to show your working?  (In particular, the later NK-9s have 5 more cost, so the n-class costs should rise slightly as well — which the old ones do but yours don't.)

Reply:
Sorted